Sources/SentrySwiftUI/SentryInternal/SentryInternal.h

```diff
@@ -0,0 +1,55 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
`m`: Please add a comment describing the pattern of exposing private classes from Sentry here. I think this is an acceptable tradeoff for now, as we don't have to access too many internal parts of Sentry.